### DISCUSSION TITLE
daily timelapse photo not reapet after finish 1 day

### DISCUSSION THREAD

**Discussion by dronevisionbtm** [gopro/labs#791] [General]
im try using daily timelapse photo, start from 07.30 finish 18.00 but next day not start capture but still have informatin will start on next day again

**Comment by dnewman-gpsw** [comment]
There are many examples of this in the discussion forum.  

For help you need to share:
- Camera model and FW version
- What script you ran  (the QR code used, preferably just the command string written below the QR Code)
- and the file MISC/qrlog.txt if the error seems weird or unexpected.

**Reply by dronevisionbtm** [reply]
thank for your respon
im using gopro 8 black ver num 02.50.71

use this qr code: mPdP>07:59<18:00!556SQ~!08:00S!1R

issued: camera not start capture after 1 day full taking photo and on camera have notif camera will start on 22hours (im check on 10am)

**Reply by dnewman-gpsw** [reply]
Log file?  Internal battery or external only power?  It seems the camera did not wake at 8am, the log will show.  This can happen, more so on older cameras.  The log file is useful to determine if anything can help.

**Reply by dronevisionbtm** [reply]
this is log of my camera
[qrlog.txt](https://github.com/gopro/labs/files/14477279/qrlog.txt)

![Untitled](https://github.com/gopro/labs/assets/161919475/4287a9cb-d852-426f-a929-a016c97eabd8)

**Reply by dronevisionbtm** [reply]
halo, any solution for me please

**Reply by dnewman-gpsw** [reply]
The camera got it wrong. An occasional failure to wake from a sleep is random on older cameras.

**Comment by dronevisionbtm** [comment]
im use external power from solar panel, how to access log file?

Sen, 4 Mar 2024 pukul 12.27 David Newman ***@***.***> menulis:

**Comment by dronevisionbtm** [comment]
oh ic, so i need to update my camera, thanks for your reply

Min, 24 Mar 2024 pukul 13.33 David Newman ***@***.***>
menulis:

**Reply by dnewman-gpsw** [reply]
This is more significantly more reliable on HERO12, and 11 pretty good too.  There are several user discussions in the forum the best configuration for long form timelapse.
